CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## UNRELEASED
 
-## Fixed
+### Added
+
+-   Configurable text cursor width. (#781)
```

Comment:
Format is:

Configurable text cursor width (#issue #pr)

Reply:
Done